mllib/src/main/scala/org/apache/spark/ml/classification/MultilayerPerceptronClassifier.scala

```diff
@@ -75,8 +75,8 @@ private[ml] trait MultilayerPerceptronParams extends PredictorParams
    * @group expertParam
    */
   final val solver: Param[String] = new Param[String](this, "solver",
-    " Allows setting the solver: minibatch gradient descent (gd) or l-bfgs. " +
-      " l-bfgs is the default one.",
+    "Allows setting the solver: minibatch gradient descent (gd) or l-bfgs. " +
+      "(Default l-bfgs)",
```

Comment:
I've seen both styles. It's not clear which is the "accepted one". I think it's this way because that's what the IDE does. Let's leave it as is though.